### PR TITLE
Add support for an optional button parameter in mouse press, release, and drag

### PR DESCRIPTION
**Pull request by austin-schick**
### What this PR does

`onMousePress` and `onMouseRelease` can now optionally take `button`, which is an integer representing the button that was pressed. `onMouseDrag` can optionally take `buttons`, which is a list of integers representing all pressed buttons.

```python3
def onMousePress(x, y, button):
    print('p', x, y, button)
    
def onMouseRelease(x, y, button):
    print('r', x, y, button)
    
def onMouseDrag(x, y, buttons):
    print('d', x, y, buttons)
```

This is a port of https://github.com/cmu-cs-academy/cs-academy/pull/4823

### CODE REVIEW

**Comment by schmave** [comment]
Looks good to me! 

If you scroll the mouse wheel, onMousePress gets called with an interesting value for button that seems to indicate the direction and amount of the scroll. Partly this seems nice, for people who want to use it. But I think it's a bit surprising in bad way for most people. Maybe only call onMousePress if `button` is <= 2?

File: foo.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from cmu_graphics import *
```
Comment (schmave) [inline foo.py:1]: Adding this file was probably unintentional

File: cmu_graphics/meta/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-1.1.22
+1.1.23
```
Comment (schmave) [inline cmu_graphics/meta/version.txt:1]: Is this still a good version number?

**Review by schmave — APPROVED** [review]
This one might have gotten lost in the shuffle a few months ago, but it looks good to me now.

File: cmu_graphics/meta/version.txt (hunk 2 above)
Comment (austin-schick) [reply cmu_graphics/meta/version.txt:1]: Yup! Master is 1.1.22 right now

**Review by schmave — APPROVED** [review]

**Pull request merged**